apps/vc-api/src/setup.ts

```diff
@@ -22,6 +22,7 @@ import { AppModule } from './app.module';
 
 async function setupApp() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors();
```

Comment:
```suggestion
  app.enableCors({ origin: true });
```

Reply:
And follow https://www.conventionalcommits.org/en/v1.0.0/ as team agree to use it in all repositories